src/applications/personalization/profile360/containers/Vet360PendingTransactionCategory.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
+
+import { refreshTransaction } from '../actions/updaters';
+
+import Vet360TransactionPending from '../components/Vet360TransactionPending';
+
+import { TRANSACTION_CATEGORY_TYPES } from '../constants/vet360';
+import { selectVet360PendingCategoryTransactions } from '../selectors';
+
+function Vet360PendingTransactionCategory({ refreshTransaction: dispatchRefreshTransaction, transactions, hasPendingCategoryTransaction, categoryType, children }) {
+  if (!hasPendingCategoryTransaction) return <div>{children}</div>;
+
+  let plural = 'email';
+  if (categoryType === TRANSACTION_CATEGORY_TYPES.PHONE) {
+    plural = 'phone numbers';
+  } else if (categoryType === TRANSACTION_CATEGORY_TYPES.ADDRESS) {
+    plural = 'addresses';
+  }
+
+  const refreshAllTransactions = () => {
+    transactions.forEach(dispatchRefreshTransaction);
+  };
+
```

Comment:
Loops through all of the pending transactions for this category-type and refreshes them